app/src/main/assets/extensions/ads/adsBackground.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a copy of https://github.com/mozilla-mobile/fenix/blob/07748f69b75144e8e247b742f66a756f999cb299/app/src/main/assets/extensions/cookies/cookiesBackground.js.